ru/lessons/advanced/gen-stage.md

```diff
@@ -0,0 +1,261 @@
+---
+layout: page
+title: GenStage
+category: advanced
+order: 11
+lang: ru
+---
+
+В этом уроке мы будем более подробно рассматривать GenStage, какую роль он выполняет, и как мы можем использовать его в наших приложениях.
+
+{% include toc.html %}
+
+## Вступление
+
+Так что же такое GenStage? Из официальной документации, это "спецификация и поток вычислений для Elixir", но что это означает для нас?
+
+Это означает, что GenStage предоставляет нам возможность определить конвейер задач, который будет выполняться независимыми шагами (или этапами) в отдельных процессах; если вы уже работали с конвейерами, тогда некоторые из этих понятий должны быть вам знакомы.
+
+Для лучшего представления того как это работает, давайте визуализируем простой поток производитель-потребитель:
+
+```
+[A] -> [B] -> [C]
+```
+
+В этом примере у нас три стадии: `A` производитель, `B` производитель-потребитель, и `C` потребитель.  `A` производит значение, которое потребляет `B`, `B` выполняет какую-то работу и возвращает значение, которое отправляется нашему потребителю `C`; роль нашей стадии важна, что мы увидим в следующей секции.
```

Comment:
I suppose `&mdash;` should be used between each of `letter — description `pair.
I'd also suggest to replace `;` with `.` again.